QtCollider/QcApplication.cpp

```diff
@@ -104,6 +104,29 @@ QcApplication::~QcApplication() {
     _mutex.unlock();
 }
 
+void QcApplication::createMenu() {
+    _mainMenu = QSharedPointer<QMenuBar>(new QMenuBar(0));
```

Comment:
The default argument for QMenuBar's constructor is nullptr, and QSharedPointer also has a more idiomatic (to modern c++) static factory function `create`, so this can be simplified to:

```cpp
_mainMenu = QSharedPointer<QMenuBar>::create();
```

Reply:
Thanks !